src/components/Sidebar/styles.ts

```diff
@@ -0,0 +1,29 @@
+
+import { makeStyles } from '@mui/styles';
+
+export default makeStyles(() => ({
+  sideBar: {
+    height: '100vh',
+    backgroundColor: '#3440e3',
+  },
+  imageLink: {
+    display: 'flex',
+    justifyContent: 'center',
+    padding: '10% 0',
+  },
+  image: {
+    width: '70%',
```

Comment:
Tip: using percentage width causes your layout to be inconsistent if the viewport width changes (e.g. phone rotating from portrait to landscape)

Reply:
For an image size on 300px (because I just created the logo on photoshop), I thought having 70% will make the image occupy 70% of the parent container?

I am not sure if having a fixed measurement is the right way, but am open to researching this (specifically for image sizes)

Reply:
You're welcome to leave it this way. Just keep in mind that it could potentially act in an unexpected way (especially for images you don't control - user inputted image)

Reply:
Okay, thank you - will remember the advice going forward